src/bentoml/_internal/cloud/deployment.py

```diff
@@ -350,37 +354,48 @@ def ensure_bento(
         if push:
             bento_api.push(bento=bento_obj, bare=bare)
         return bento_obj
-    elif bento:
-        bento = Tag.from_taglike(bento)
-        try:
-            bento_obj = _bento_store.get(bento)
-        except NotFound:
-            bento_obj = None
+    assert bento is not None
```

Comment:
I don't think we should have `assert` in code like this no?

ig we don't use `-O3` anytime soon, so it doesn't affect anything

Reply:
This is only to make the type checker happy